src/main/kotlin/com/exactpro/th2/common/schema/factory/ExactproMetaInf.kt

```diff
@@ -106,10 +106,21 @@ internal class ExactproMetaInf(
                         attributes.getValue(Attributes.Name.IMPLEMENTATION_VERSION)
                     )
                 }
-            } catch (e: IOException) {
+            } catch (e: Exception) {
                 K_LOGGER.warn(e) { "Manifest '$manifestUrl' loading failure" }
                 return null
             }
         }
+
+        private fun URL.toPath(): Path = toURI().run {
+            // the code below are added to provide windows compatibility
+            when (scheme) {
+                "file" -> Path.of(this)
+                "jar" -> Path.of(schemeSpecificPart
+                    .substringBefore("!")
```

Comment:
`!` is [an allowed character](https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file) for file name in Windows. Because of that it looks like this code might not work correctly if there is a `!` in the file path